datajunction-server/datajunction_server/api/namespaces.py

```diff
@@ -144,12 +144,21 @@ async def list_nodes_in_namespace(
         default=None,
         description="Filter the list of nodes to this type",
     ),
+    with_edited_by: bool = Query(
+        default=False,
+        description="Whether to include a list of users who edited each node",
```

Comment:
Could you pls clarify a use case for this param? I can seem to put a finger on a good one...

Reply:
From this thread: https://github.com/DataJunction/dj/pull/1126#discussion_r1695571189